### ISSUE TITLE
Tf 2.1, cause system crash and reboot

### ISSUE THREAD

**Issue by pykeras** [tensorflow/tensorflow#37216]
**System information** 
* POP_OS 19.10, Kernel: 5.3.0-7629-generic
* Tensorflow: installed using `pip`, `__version__` 2.1.0
* Nvidia_driver: 440.44, Cuda : 10.2 
* GPU: Nvidia GTX 1080Ti , Memory : 11GB

**Describe the current behavior**

Hi 
Just tried to run an example on `TF2.1`, every time right after starting training the system will crash immediately and will reboot (because of crash reboot there is no useful log available on `/var/log/`)

So decide to change my code to a simpler version and using the cifar10 dataset to check that, again system crash happend. this time I switched to `TF1.13.1` and run the same code and no problem, system work properly and at last, print the result.

even tried on other versions `TF1.14`, `1.15` there is no problem (all versions are GPU support)

**Standalone code to reproduce the issue** 
[sample code](https://github.com/pykeras/bugReport/blob/master/sample.py)

**Comment by ravikyram** [comment]
@pykeras 
I have tried on colab with TF version 2.1.0  and i am not seeing any issue. Please, find the gist [here](https://colab.sandbox.google.com/gist/ravikyram/2f99c48131c5e08cf99534f0515105ac/untitled692.ipynb). Thanks!

**Comment by pykeras** [comment]
@ravikyram 

Thanks for responde

I tried different things on my OS, found that the problem is `BatchNormalization()` layer, every time when I remove these layers from codes (even the sample I shared or my other codes), there is no problem at all

but right after adding the BatchNorm layers, the system will crash

**Comment by pykeras** [comment]
Update: 

Just tried another thing, using another OS with same hardware solved the problem
so I don't know what cause the problem, but sure this is not only about TF2.1 

Because of this I think should close the issue, Thanks

**Comment by liuxingbaoyu** [comment]
excuse me
This may be related to [https://github.com/tensorflow/tensorflow/issues/36072](https://github.com/tensorflow/tensorflow/issues/36072)

**Comment by raghavendraLhci** [comment]
My system didn't crash but hanged after a few batches of first epoch.. At first i felt it is related to GPU temperature like someone mentioned above. But just updating the driver resolved it.. Thanks to the one who mentioned it.. Saved my life..!!!!
